packages/common/src/error.ts

```diff
@@ -24,18 +25,20 @@ type BaseErrorOptions<ContextType> = {
 	silent?: boolean;
 };
 
-interface BaseContextParams {
+export interface BaseContextParams {
 	// biome-ignore lint/suspicious/noExplicitAny: TODO remove any
 	[key: string]: any;
```

Comment:
while we're here can we change this to 

[key: string]: unknown;

and then drop the biome-ignore

Reply:
done